themes/survey/bootswatch/css/variations/cosmos.min.css

```diff
@@ -1,4 +1,4 @@
-@import url("https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700");/*!
+@import url("../../../../../assets/fonts/source-sans-pro.css");/*!
```

Comment:
Yep … really don't see how to do it differently …
Don't really like it, but no idea on a cleaner way to do …